int16 cell_id
string frame_id
float32 voltage
